app/views/idv/phone/new.html.erb

```diff
@@ -38,7 +38,16 @@
 </ul>
 
 <p>
-  <%= t('idv.messages.phone.final_note_html') %>
+    <%= link_to(
+          t('idv.troubleshooting.options.learn_more_verify_by_phone'),
+          help_center_redirect_url(
+            category: 'verify-your-identity',
+            article: 'phone-number',
+            flow: :idv,
+            step: :phone,
+          ),
+          class: 'usa-link usa-link--external',
```

Comment:
Do we intend for this link to be opened in a new window? I believe that's conventional for any link going to an external domain, or at least especially because it may disrupt the user's progress in the flow to be taken away in the same tab.

If so, we can use the `new_window_link_to` help and drop the explicit class reference.

```suggestion
    <%= new_window_link_to(
          t('idv.troubleshooting.options.learn_more_verify_by_phone'),
          help_center_redirect_url(
            category: 'verify-your-identity',
            article: 'phone-number',
            flow: :idv,
            step: :phone,
          ),
```

Reply:
Thanks, didn't even know about `new_window_link_to`